### PR TITLE
Eperez amps

### PR DESCRIPTION
**Pull request by enriquepablo**
Different local dbs for different services

### CODE REVIEW

File: src/eduid_webapp/email/validators.py [hunk 1]
```diff
@@ -32,12 +32,12 @@
...
 from marshmallow import ValidationError
-from flask import request
-from eduid_common.api.utils import get_dashboard_user
+from flask import request, current_app
+from eduid_common.api.utils import get_user
 
 
 def validate_email(email):
-    user = get_dashboard_user()
+    user = get_user(current_app.email_proofing_userdb)
```
Comment (johanlundberg) [inline src/eduid_webapp/email/validators.py]: Shouldn't user here be the authorized user returned from the central db?

I don't think you ever need to read the user from the proofing_userdb, only write it and sync to the central db.

File: src/eduid_webapp/email/views.py [hunk 2]
```diff
@@ -150,7 +150,7 @@ def post_primary(user, email, csrf_token):
 
 
 def _steal_mail(email):
```
Comment (johanlundberg) [inline src/eduid_webapp/email/views.py]: The "stealing" is now done before a write to the central db, https://github.com/SUNET/eduid-am/blob/master/eduid_am/consistency_checks.py.

File: src/eduid_webapp/phone/helpers.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+from flask import current_app, session
+
+from eduid_userdb import User
+from eduid_userdb.proofing import ProofingUser
+from eduid_userdb.exceptions import UserDoesNotExist, MultipleUsersReturned
+from eduid_common.api.utils import retrieve_modified_ts, get_user
+from eduid_common.api.exceptions import ApiException
+
+
+def require_user(f):
```
Comment (johanlundberg) [inline src/eduid_webapp/phone/helpers.py]: Same comments as for email applies here.

File: src/eduid_webapp/email/helpers.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+from flask import current_app, session
+
+from eduid_userdb import User
+from eduid_userdb.proofing import ProofingUser
+from eduid_userdb.exceptions import UserDoesNotExist, MultipleUsersReturned
+from eduid_common.api.utils import retrieve_modified_ts, get_user
+from eduid_common.api.exceptions import ApiException
+
+
+def require_user(f):
```
Comment (enriquepablo) [inline src/eduid_webapp/email/helpers.py]: It is fetched from the central db `get_user` only uses the local db to update the modified_ts attr.

I don't use the one from eduid_common.api.decorators b/c that one is not using retrieve_modified_ts. But I was saying in the mail, retrieve_modified_ts makes little sense now...

File: src/eduid_webapp/email/views.py (hunk 2 above)
Comment (enriquepablo) [inline src/eduid_webapp/email/views.py]: removing it

**Pull request merged**